feature/src/main/java/com/teamdontbe/feature/login/SignUpProfileActivity.kt

```diff
@@ -0,0 +1,44 @@
+package com.teamdontbe.feature.login
+
+import androidx.activity.viewModels
+import com.teamdontbe.core_ui.base.BindingActivity
+import com.teamdontbe.core_ui.util.context.colorOf
+import com.teamdontbe.feature.R
+import com.teamdontbe.feature.databinding.ActivitySignUpProfileBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class SignUpProfileActivity :
+    BindingActivity<ActivitySignUpProfileBinding>(R.layout.activity_sign_up_profile) {
+
+    private val viewModel by viewModels<SignUpProfileViewModel>()
+
+    override fun initView() {
+        binding.vm = viewModel
+        initBtnSignUpProfileClickListner()
+        initNickNameExistObserving()
+    }
+
+    private fun initBtnSignUpProfileClickListner() {
+        binding.btnSignUpProfileDoubleCheck.setOnClickListener {
+            val nickName = binding.etSignUpAgreeText.text.toString()
+            viewModel.validateNickName(nickName)
```

Comment:
P1 : 
```suggestion
            viewModel.validateNickName(binding.etSignUpAgreeText.text.toString())
```
변수 생성 안해도 되어 보입니다-! 오히려 xml id를 et_sign_up_agree_nickname으로 직관적으로 수정하면 좋을듯!

Reply:
[22c39d2](https://github.com/TeamDon-tBe/ANDROID/pull/52/commits/22c39d22fbb084c36e0635d13fe6e424e493464d)

Reply:
[9794671](https://github.com/TeamDon-tBe/ANDROID/pull/52/commits/97946718707333540078552f5b2f00cf6f69e5fa)